src/github.com/matrix-org/go-neb/database/schema.go

```diff
@@ -262,6 +262,35 @@ func selectRealmTxn(txn *sql.Tx, realmID string) (types.AuthRealm, error) {
 	return realm, nil
 }
 
+const selectRealmsByTypeSQL = `
+SELECT realm_id, realm_json FROM auth_realms WHERE realm_type = $1
```

Comment:
The code doesn't rely on order currently. The API does not specify an order. If we need an order for whatever reason, then we can chuck in an `ORDER BY` I think.